pkg/openwsn/include/openwsn_log.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_openwsn
+ * @{
+ *
+ * @file
+ * @brief       System logging header OpenWSN definitions
+ *
+ * This header implements the same functionality as log.h but avoiding
+ * macro redefinitions with OpenWSN LOG_% functions.
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef LOG_RIOT_H
```

Comment:
this needs fixing (`OPENWSN_LOG_RIOT_H`) here and at the bottom

Reply:
please squash right in

Reply:
fixed